packages/wasm/src/keys.test.ts

```diff
@@ -1,17 +1,87 @@
 import { describe, expect, it, vi } from 'vitest';
-import { generateSpendKey } from './keys';
+import {
+  generateSpendKey,
+  getAddressByIndex,
+  getFullViewingKey,
+  getShortAddressByIndex,
+} from './keys';
 
 // Replace the wasm-pack import with the nodejs version so tests can run
 vi.mock('@penumbra-zone/wasm-bundler', () => vi.importActual('@penumbra-zone/wasm-nodejs'));
 
 describe('keys', () => {
+  // don't use this seed phrase for personal use
+  const seedPhrase =
+    'mushroom hole price march grid pepper goat position print gaze dignity music milk surge file casino unveil betray observe ethics image topic sphere clap';
+
   describe('generateSpendKey()', () => {
     it('does not raise zod validation error', () => {
-      const seedPhrase =
-        'mushroom hole price march grid pepper goat position print gaze dignity music milk surge file casino unveil betray observe ethics image topic sphere clap';
       expect(() => {
         generateSpendKey(seedPhrase);
       }).not.toThrow();
     });
+
+    it('equal to penumbraspendkey1fcesjzswatt6g6hkr0jvu6svc8xl7zq3yzzxrnkaz470nvc039ts8c3e4d', () => {
+      const spendKey = generateSpendKey(seedPhrase);
+      expect(spendKey).equal(
+        'penumbraspendkey1fcesjzswatt6g6hkr0jvu6svc8xl7zq3yzzxrnkaz470nvc039ts8c3e4d',
+      );
+    });
+  });
```

Comment:
We do not need to test implementation in these tests. We just need to validate types.